spalp.module

```diff
@@ -50,12 +53,16 @@ function spalp_theme() {
  * @return array
  *   An array of possible key and value options.
  *
- * @see options_allowed_values()
  */
-function spalp_get_app_ids(FieldStorageConfig $definition, ContentEntityInterface $entity = NULL, $cacheable) {
-
+function spalp_get_app_ids(FieldStorageConfig $definition, ContentEntityInterface $entity = NULL, $cacheable) {  
```

Comment:
Consider making this into a service and not a function in a module file.
`function spalp_get_app_ids`

see (LOGIC) section in:
https://www.drupal.org/project/spalp/issues/2993322